content/blog/2020-04-tidymodels-org/index.Rmarkdown

```diff
@@ -0,0 +1,52 @@
+---
+title: Announcing tidymodels.org
+slug: tidymodels-org
+description: >
+    We're thrilled to announce the release of tidymodels.org! The site is meant
+    to be a central location for learning and using the tidymodels packages. 
+date: 2020-04-17
+author: Max Kuhn
+photo:
+  url: https://unsplash.com/photos/Hli3R6LKibo
+  author: Adi Goldstein
+categories: [learn]
+tags:
+  - tidymodels
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px")
+```
+
+We're thrilled to announce the release of [`tidymodels.org`](https://www.tidymodels.org)! The site is meant to be a central location for learning and using the tidymodels packages. The site is a new take on a familiar look:
+
+![](tidymodels-front-page.jpg)
+
+This site has a different organization than its tidyverse sibling. 
+
+The [**Getting Started** landing site](https://www.tidymodels.org/start/) has a series of five articles that are aimed at readers who have little to no experience with the tidymodels packages. Each article focuses on a specific task (e.g. building a model). The last article shows a small case study that brings all of the other elements together. 
```

Comment:
```suggestion
The [**Getting Started**](https://www.tidymodels.org/start/) page has a series of five articles that are aimed at readers who have little to no experience with the tidymodels packages. Each article focuses on a specific task (e.g. building a model), and the last article walks through a case study that brings all of the other elements together. 
```

Reply:
For consistency, "Getting Started" --> "Get started" (then matches top navbar)